lib/image-stream/utils.js

```diff
@@ -41,10 +41,6 @@ exports.getArchiveMimeType = (filename) => {
   const mimeType = mime.lookup(filename);
 
   if (mimeType) {
-    if (mimeType === 'application/x-iso9660-image') {
-      return Bluebird.resolve(MIME_TYPE_RAW_IMAGE);
```

Comment:
I't still used in the fallback below – should we just pass the literal string there, or keep the var?